src/ARCtrl/ARCtrl.Javascript.fsproj

```diff
@@ -0,0 +1,90 @@
+    <Description>Library for management of Annotated Research Contexts (ARCs) using an in-memory representation and runtimer agnostic contract systems.</Description>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageIcon>logo.png</PackageIcon>
+    <PackageTags>ARC F# FSharp dotnet .Net bioinformatics biology fable-library datascience dataplant nfdi metadata</PackageTags>
+    <PackageProjectUrl>https://github.com/nfdi4plants/ARCtrl</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/nfdi4plants/ARCtrl</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Json\Decode.fs" />
+    <Compile Include="Json\Encode.fs" />
+    <Compile Include="Json\Comment.fs" />
+    <Compile Include="Json\OntologyAnnotation.fs" />
+    <Compile Include="Json\OntologySourceReference.fs" />
+    <Compile Include="Json\DataFile.fs" />
+    <Compile Include="Json\Data.fs" />
+    <Compile Include="Json\Person.fs" />
+    <Compile Include="Json\Publication.fs" />
+    <Compile Include="Json\Process\Value.fs" />
+    <Compile Include="Json\Process\Factor.fs" />
+    <Compile Include="Json\Process\FactorValue.fs" />
+    <Compile Include="Json\Process\ProtocolParameter.fs" />
+    <Compile Include="Json\Process\MaterialType.fs" />
+    <Compile Include="Json\Process\MaterialAttribute.fs" />
+    <Compile Include="Json\Process\Component.fs" />
+    <Compile Include="Json\Process\Protocol.fs" />
+    <Compile Include="Json\Process\MaterialAttributeValue.fs" />
+    <Compile Include="Json\Process\Material.fs" />
+    <Compile Include="Json\Process\Source.fs" />
+    <Compile Include="Json\Process\Sample.fs" />
+    <Compile Include="Json\Process\ProcessParameterValue.fs" />
+    <Compile Include="Json\Process\ProcessInput.fs" />
+    <Compile Include="Json\Process\ProcessOutput.fs" />
+    <Compile Include="Json\Process\Process.fs" />
+    <Compile Include="Json\Process\ProcessSequence.fs" />
+    <Compile Include="Json\Process\StudyMaterials.fs" />
+    <Compile Include="Json\Table\Compression.fs" />
+    <Compile Include="Json\Table\CompositeCell.fs" />
+    <Compile Include="Json\Table\IOType.fs" />
+    <Compile Include="Json\Table\CompositeHeader.fs" />
+    <Compile Include="Json\Table\ArcTable.fs" />
+    <Compile Include="Json\Table\Templates.fs" />
+    <Compile Include="Json\DataMap\DataMap.fs" />
+    <Compile Include="Json\Assay.fs" />
+    <Compile Include="Json\Study.fs" />
+    <Compile Include="Json\Investigation.fs" />
+    <Compile Include="Json\ROCrateObject.fs" />
+    <Compile Include="Json\ARC.fs" />
+    <Compile Include="WebRequest\WebRequest.Node.fs" />
+    <Compile Include="WebRequest\WebRequest.fs" />
+    <Compile Include="Template.Web.fs" />
+    <Compile Include="ARC.fs" />
+    <Compile Include="Json.fs" />
+    <Compile Include="Xlsx.fs" />
+    <None Include="README.md" />
+  </ItemGroup>
```

Comment:
If you want to avoid duplication of the code you can move it to `ARCTrl.Common.props` and include it in your different project.

Some of the properties (like Authors, License) can even be moved to the top level `Directory.Build.props`

Reply:
Will try, thanks!